packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -235,6 +237,23 @@ export class Account {
   async utxosToAccount (payload: UtxosToAccountPayload, utxos: UtxosToAccountUTXO[] = []): Promise<string> {
     return await this.client.call('utxostoaccount', [payload, utxos], 'number')
   }
+
+  /**
+   * Returns count of account history
+   *
+   * @param {OwnerType | string} [owner= OwnerType.MINE] single account ID (CScript or address) or reserved words 'mine' to list history count for all owned accounts or 'all' to list whole DB
+   * @param {AccountHistoryCountOptions} [options]
+   * @param {boolean} [options.no_rewards]  Filter out rewards
+   * @param {string} [options.token]  Filter by token
+   * @param {string} [options.txtype]  Filter by transaction type, supported letter from 'CRTMNnpuslrUbBG'
```

Comment:
Since you are working on this, we need to refactor this to follow how OwnerType is done now. I have assigned you https://github.com/DeFiCh/jellyfish/issues/252#issuecomment-843766560 you can do it in this PR.

Instead of `txtype?: string` to `txtype?: TxType | string`

Reply:
Implemented `TxType enum` referencing #108.